### DISCUSSION TITLE
how to start smplayer without loading last file/playlist?

### DISCUSSION THREAD

**Discussion by viper73** [smplayer-dev/smplayer#487] [Q&A] [answered]
Hello,

i use smplayer sometimes to show streams from network. Then i load a playlist with my streams. On the next start of smplayer the player automatically loads this last playlist again (or does not empty it). I do not want that, i want to start every time without my last playlist or without playing the last file i watched. So i want smplayer to to start every time "empty".

How can i do that? I can not find an option to empty the playlist after closing smplayer or start smplayer without loading last file/playlist.

Greets, Viper73

**Comment by smplayer-dev** [comment, marked as answer]
Turn off the option `save a copy of the playlist on exit` in preferences -> playlist -> misc.

**Reply by viper73** [reply]
Great! Thank you.
